code/__DEFINES/mobs.dm

```diff
@@ -479,10 +479,16 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define HUNTER_SNEAKATTACK_MULTI_RECOVER_DELAY	10
 
 //Ravager defines:
-#define RAV_CHARGESPEED					2
-#define RAV_CHARGESTRENGTH				2
-#define RAV_CHARGEDISTANCE				4
-#define RAV_CHARGE_TYPE					3
+#define RAVAGER_CHARGESPEED					2
+#define RAVAGER_CHARGESTRENGTH				2
+#define RAVAGER_CHARGEDISTANCE				4
+#define RAVAGER_CHARGE_TYPE					3
+
+#define RAVAGER_ENDURE_DURATION				20 SECONDS
+#define RAVAGER_ENDURE_WARNING				0.66
+
+#define RAVAGER_RAGE_DURATION				10 SECONDS
+#define RAVAGER_RAGE_WARNING				0.7
```

Comment:
Are all of these needed anymore?

specifically `RAVAGER_CHARGE_TYPE`

Reply:
Not really touching on charge but I'll take a look.